content/millennium/r4/encounters/encounter.md

```diff
@@ -97,3 +97,56 @@ The common [errors] and [OperationOutcomes] may be returned.
 [`_count`]: http://hl7.org/fhir/R4/search.html#count
 [errors]: ../../#client-errors
 [OperationOutcomes]: ../../#operation-outcomes
+
+## Create
+
+Create an individual Encounter.
+
+    POST /Encounter
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', 'Content-Type': 'application/json+fhir'} %>
```

Comment:
Should be application/fhir+json for content-type

Reply:
added: https://github.com/cerner/fhir.cerner.com/pull/239/commits/b320b5dc6baee634a654bc534bd79229818fa6fe

Reply:
Eddy added this to the headers method https://github.com/cerner/fhir.cerner.com/commit/761f781e1eb58169b9b02d2530719b468b68a24d#diff-7718c11ed130ac00a9854e0b089f74b4L27

Reply:
It looks like that doesn't apply here because default_headers_r4 only pulls in accept type and not content type.